demo/demo.js

```diff
@@ -7,75 +104,44 @@ $(function () {
   var url = 'ws://localhost:9000/ws';
 
   // create the session
-  var sessionPromise = vcs.createSession(url);
+  sessionPromise = vcs.createSession(url);
 
   sessionPromise.catch(() => {
     console.log('Could not connect to ' + url);
   });
 
-  // create the canvas
   var canvasPromise = sessionPromise.then(function (session) {
-    return vcs.init(document.getElementById('vcs-plot-1'), session);
-  });
-
-  // create the graphics method
-  var gmPromise = canvasPromise.then(function (canvas) {
-    // create a graphics method
-    return canvas.create('isofill', '¯\_(ツ)_/¯')
-  });
-
-  // get the data object for the clt variable
-  var cltPromise = sessionPromise.then(function (session) {
-    return session.files();
-  }).then(function (files) {
-    // print the list of files
-    console.log(files.map((f) => f.fileName));
-
-    // get the file object for a netcdf variable
-    var nc = files.filter(function (f) { return f.fileName.match(/\.nc$/); })[0];
-
-    if (!nc) {
-      console.log('No netcdf variables found');
-      return;
-    }
-
-    // get a list of variables
-    return nc.variables();
-  }).then(function (variables) {
-    console.log(variables);
+    return session.init(document.getElementById('vcs-isofill'));
   });
-
+    
   // generate the plot when all of the promises resolve
-  Promise.all([
-   canvasPromise, gmPromise, cltPromise
-  ]).then(function (arg) {
-    var canvas = arg[0];
-    var gm = arg[1];
-    var clt = arg[2];
-
-    canvas.plot(clt, gm, 'default', 'vtkweb');
+  canvasPromise.then(function (canvas) {
+    var dataSpec = {
+      file: 'coads_climatology.nc',
```

Comment:
Here is the file. Not sure why this was used in the original cdat_web.
[coads_climatology.zip](https://github.com/UV-CDAT/vcs-js/files/527542/coads_climatology.zip)